news/_posts/2020-06-12-product-features-methodology-at-prestashop.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Product development at PrestaShop"
+subtitle: "What happens in the team when working on new features."
+date: 2020-06-12 9:00:00
+authors: [SamuelPires]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-lab
+tags:
+- design
+- feature
+- process
+- product
+---
+
+In March 2020, the Core product team at PrestaShop, in charge of building and prioritizing new features, decided to work with a new tool: Productboard.
+
+You can learn more in this dedicated article about [how to use Productboard to share feedback on features the team is considering working on]().
+
+The team thought it was the right occasion to talk more about the product development at PrestaShop, the steps, the roles and responsibilities and what happens behind the scenes before a new feature is added to PrestaShop.
+
+
+## PrestaShop’s product development cycle
+
+### The Core product team
+At the Prestashop company, a team is dedicated to maintaining the PrestaShop open source project: the Core team.
+Its mission is to make sure the software is stable, usable, delivers value to its users and follows the industry standards.
+All this requires several people and several skills. Today the Core team is composed of developers, QA engineers, designers and product managers.
+Editing and maintaining a software like PrestaShop means fixing bugs but also adding improvements and new features.
+
+While bug fixing is most of the time handled directly by developers (from the Core team and the community 😉), features design and development are much more complex. Why? Because the goal of any feature is to help users achieve something, **behind every feature there should be a user need**.
```

Comment:
Yes it's quite "messy". I think we could talk about the different types of skills when first talking about the Core team (see 2 blocks above).

Then about the differences between bugs and features, I think what we can say is that most of the time the functional solution to fix a bug is obvious, while adding a new feature requires indepth search and conception as the main goal is to solve a user problem. What's the problem ? How many users have the same problem ? How painful is it ? How often do users have this problem ?

Reply:
```suggestion
Fixing a bug does not require the same skills that defining a new feature. Most of the time, the functional scope of a bug is obvious: the 'save' button does not work, let's fix it so that it can save. Adding a feature is a bit more complex: in-depth search must be done to specify the functional scope.

Why so? Because **the goal of any feature is to solve a problem**. So we need to answer the following questions get a better definition of the issue: what exactly is the problem? how many users encounter it? how painful is it? is there any workaround?
```